src/util/DataWriterLE.ts

```diff
@@ -4,34 +4,84 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+function toNumber(value: bigint | number): number {
```

Comment:
Can these be static under some utility class or namespace to simplify imports?

Reply:
Already done in a few PRs later in the chain :-) (I created Number.ts that contains min / max and those functions